docs/business.yaml

```diff
@@ -3,8 +3,33 @@ info:
   title: businesses
   version: '2.0'
   description: |-
-    Businesses provideas all of the services needed to manage legal entities.
-    This encompasses all types of businesses authorized to do business by the Registrar.
+    The Business Registry is a centralized electronic register of all business in British Columbia, This encompasses all types of businesses authorized to do business by the Registrar. The Business API provides a secure means of integrating your business systems with the Registry.
+    <br/>
+    <br/>This API supports the following filings:
+    - Incorporation Application
+    - Amalgamation Regular
+    - Amalgamation Short Form Horizontal 
+    - Amalgamation Short Form Vertical 
+    - Continuation In 
+    - Registration 
+    - AGM Extension Information 
+    - AGM Location Change Information 
+    - Alteration 
+    - Annual Report 
+    - Change of Address
+    - Change of Directors
+    - Change of Entity Name
+    - Change of Registration 
+    - Consent Continuation Out 
+    - Conversion ( to change the business type of an organization eg: convert from a Member Funded Society to a BC Company)
+    - Court Order Information
+    - Correction
+    - Dissolution
+    - Registration (sole proprietorship, general partnership)
```

Comment:
And is it better to also capitalize first letters for SP/GP?

Reply:
sounds good